routes/auth/claim.ts

```diff
@@ -33,10 +33,10 @@ export async function mintClaimedKeyId(
 			authMethodPubkey: "0x",
 		});
 		console.info("claimed key id: transaction hash (request id): ", {
-			requestId: mintTx.hash
+			requestId: mintTx.transactionHash,
```

Comment:
if `.hash` was working before shouldn't we try to return either/or result?

Reply:
why do we need this change? arent both hash values identical for the tx hash?

Reply:
@joshLong145 .hash was actually undefined when I was using it. Was it the same for you?

Reply:
yes after testing again i see `hash` was undefined. merging this in as the type change is required after calling `tx.wait()` on the returned transaction.